Node/node-example-produce.js

```diff
@@ -1,11 +1,3 @@
-<<<<<<< HEAD
-import { StreamrClient } from 'streamr-client';
-
-// Create the client and supply either an API key or an Ethereum private key to authenticate
-const client = new StreamrClient({
-    auth: {
-        privateKey: 'ETHEREUM-PRIVATE-KEY',
-=======
 import StreamrClient from 'streamr-client';
 
 // REPLACE THIS WITH YOUR OWN PRIVATE KEY
```

Comment:
This now checks to see if you have replaced the private key with your own or not